virttest/utils_net.py

```
@@ -3961,8 +3968,9 @@ def get_windows_nic_attribute(
         )
         raise exceptions.TestError(err_msg)
     lines = [l.strip() for l in out.splitlines() if l.strip()]
```

Ruff (0.14.8)

Ambiguous variable name: `l`

(E741)